lib/src/test/java/com/auth0/jwt/algorithms/HMACAlgorithmTest.java

```diff
@@ -53,19 +54,16 @@
     @Test
     public void shouldGetStringBytes() throws Exception {
         String text = "abcdef123456!@#$%^";
-        byte[] expectedBytes = text.getBytes("UTF-8");
+        byte[] expectedBytes = text.getBytes(StandardCharsets.UTF_8.name());
         assertTrue(Arrays.equals(expectedBytes, HMACAlgorithm.getSecretBytes(text)));
     }
 
     @Test
     public void shouldPassHMAC256Verification() throws Exception {
         String token = "eyJhbGciOiJIUzI1NiIsImN0eSI6IkpXVCJ9.eyJpc3MiOiJhdXRoMCJ9.mZ0m_N1J4PgeqWmi903JuUoDRZDBPB7HwkS4nVyWH1M";
         Algorithm algorithmString = Algorithm.HMAC256("secret");
-        Algorithm algorithmBytes = Algorithm.HMAC256("secret".getBytes(StandardCharsets.UTF_8));
         JWT jwt = JWT.require(algorithmString).withIssuer("auth0").build();
```

Comment:
You changed all the tests to use `JWT.require` instead. Please roll back all these changes. Those tests are already available [here](https://github.com/auth0/java-jwt/blob/master/lib/src/test/java/com/auth0/jwt/JWTTest.java).
This comment applies to all `XAlgorithmTest` classes